### ISSUE TITLE
Christmas Lights texture Issue

### ISSUE THREAD

**Issue by Paeadams** [MehVahdJukaar/SnowySpirit#27]
<img width="1440" alt="Screenshot 2022-11-11 at 12 46 27 AM" src="https://user-images.githubusercontent.com/109878274/201229447-0c761bb5-1f4d-4490-bc4a-33f366c7db60.png">
The Leaves on the image are not visible after puting christmas lights on them!

**Comment by MehVahdJukaar** [comment]
Do you use ootifine or sodium?

**Comment by Paeadams** [comment]
Sodium yes is that the issue?

**Comment by Paeadams** [comment]
If yes, is there a way to use it without his issue happening?

**Comment by MehVahdJukaar** [comment]
It is if you don't have indium
 Sodium in its own breaks fabric rendering api

**Comment by Paeadams** [comment]
So if I download Indium the problem will solve?

**Comment by MehVahdJukaar** [comment]
Yes if you use sodium with mods you always want to install indium as sodium is not completely compatible with fabric
